armory/datasets/package.py

```diff
@@ -18,16 +18,28 @@ def package(
     """
     Package a built dataset as .tar.gz, return path
     """
-    version, data_dir, built_data_dir, subdir = build.build_info(
+    version, data_dir, built_data_dir, subdir, builder_configs = build.build_info(
         name, version=version, data_dir=data_dir
     )
 
+    # print(builder_configs)
     data_dir = Path(data_dir)
-    expected_dir = data_dir / name / version
-    if not expected_dir.is_dir():
-        raise FileNotFoundError(f"Dataset {name} not found at {expected_dir}")
-    tar_full_filepath = common.get_cache_dataset_path(name, version)
 
+    if not builder_configs:
+        tar_list = [str(Path(name) / version)]
+    else:
+        # metadata.json contains default_config_name for the given dataset
```

Comment:
this comment about `metadata.json` isn't clearly tied to the line below it (I know from reviewing that this json exists inside `.config/`, but the comment doesn't make that clear). Without this context it's unclear what it's trying to say

Reply:
check 23c35782

Reply:
resolved with b6cd933f